types/src/crypto_proxies.rs

```diff
@@ -92,9 +91,17 @@ pub type LedgerInfoWithSignatures = RawLedgerInfoWithSignatures<Ed25519Signature
 pub type ValidatorInfo = RawValidatorInfo<Ed25519PublicKey>;
 pub type ValidatorVerifier = RawValidatorVerifier<Ed25519PublicKey>;
 pub type ValidatorSigner = RawValidatorSigner<Ed25519PrivateKey>;
-pub type ValidatorChangeEventWithProof = RawValidatorChangeEventWithProof<Ed25519Signature>;
 pub type ValidatorPublicKeys = RawValidatorPublicKeys<Ed25519PublicKey>;
 pub type ValidatorSet = RawValidatorSet<Ed25519PublicKey>;
+pub use crate::validator_change::ValidatorChangeEventWithProof;
+
+#[derive(Clone)]
+/// EpochInfo represents the highest trusted validator set: it is updated every time we're crossing
+/// the epoch boundary.
```

Comment:
Not sure this is accurate.  EpochInfo represents the ValidatorVerifier that is used for validating signatures from a particular epoch.

Reply:
lol, copy-paste

Reply:
Well, it used to be the case for state sync, and I think it's still the case for the client, although it's the property of a variable and not the property of a struct. My bad :)